src/main/java/interface_adapters/tui/controllers/RecipeNoteOperation.java

```diff
@@ -39,7 +34,7 @@ public void run() {
 
     @Override
     public String getDescription() {
-        return "Create, list, edit, and delete notes";
+        return "Create, list, edit, and delete note";
```

Comment:
The grammar is incorrect here, it should be "Create, list, edit, and delete notes"